CHAOSScon/2020EU/schedule.md

```diff
@@ -1,3 +1,305 @@
-# Schedule
+# Schedule: January 31, 2020
 
-TBD
+<table>
+<thead>
+<!-- Header -->
+<tr valign=top>
+<th><h3 align=center>Time</h3>
+<th colspan=2><h3 align=center>Session</h3>
+<!-- Body -->
+
+<tbody>
+
+<!-- 9:00 - 9:30 --><!--CONFIRMED--><!--INTERIM-->
+<tr valign=top>
+<th align=right><time>9:00</time>
+<th colspan=2><!--Interim-->
+Registration & Networking
+
+<!-- 9:30 - 9:50 --><!--UNCONFIRMED--><!--KEYNOTE-->
+<tr valign=top>
+<th align=right><time>9:30</time>
+<td align=center colspan=2><small><!--Session-->
+<b><samp style="color:#9999">TBA</samp></b><br/>
+<cite><author hidden><samp style="color:#9999">TBA</samp></author></cite>
+<td hidden align=left><small><!--Session-->
+<td hidden align=center><small>
+
+<!-- [<kbd>Slides</kbd>]() -->
+
+<!-- [<kbd>Video</kbd>]() -->
+
+<!-- 9:50 - 10:20 --><!--UNCONFIRMED--><!--KEYNOTE-->
+<tr valign=top>
+<th align=right><time>9:50</time>
+<td align=center colspan=2><small><!--Session-->
+<b><samp style="color:#9999">TBA</samp></b><br/>
+<cite><author hidden><samp style="color:#9999">TBA</samp></author></cite>
+<td hidden align=left><small><!--Session-->
+<td hidden align=center><small>
+
+<!-- [<kbd>Slides</kbd>]() -->
+
+<!-- [<kbd>Video</kbd>]() -->
+
+<!-- 10:20 - 10:40 --><!--CONFIRMED-->
+<tr valign=top>
+<th align=right><time>10:20</time>
+<td align=left><small><!--Session-->
+<b>Heat, Light, and Love: Telling Twitter's Open Source Program Office's Story</b><br/>
+<cite><author>Sean G. & Remy DeC</author></cite>
```

Comment:
line 50 and others, Please use the full Speaker names that match the columns O "Primary Speaker Full Name" and Q "Co-speakers".